modules/augain/augain.c

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file augain.c  Audio gain
+ *
+ * Copyright (C) 2024 Juha Heinanen
+ */
+
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+#include <stdlib.h>
```

Comment:
System headers like `stdlib.h` should be included before others:

https://github.com/baresip/re/wiki/Development-(re-usage)#include-headers